modules/ROOT/pages/anypoint-monitoring-visualizer.adoc

```diff
@@ -0,0 +1,197 @@
+InfluxDB requires a valid license to operate, which should have been provided at the time of purchase of Anypoint Monitoring.
+
+Anypoint Monitoring requires a Kubernetes secret to be present in the cluster before all Anypoint Monitoring/Visualizer charts are deployed.
+
+The following command needs to be run in order to create the required secret in the default namespace:
+
+
+```
+kubectl create secret generic influxdb-license --from-literal=license={license}
+```
+
+Note that InfluxDB license might be a JSON file in which case we only need the `key` value in that secret.
+
+If no secret is given, InfluxDB will fail with the following message
+
+```
+ts=ZULU_TIME lvl=error msg="LICENSE IS INVALID OR CANNOT BE FOUND" log_id=LOG_ID service=licensing error=ERROR_MESSAGE
+```
+
+If that is the case, the secret will need to be created and InfluxDB chart redeployed. Before redeploying InfluxDB, all *pvc*, *pv* and *jobs* (named `dias-prov-k8s-am-influxdb-comp-bootstrap`) will need to be deleted.
+
+
+
+== Restore/Backup
+
+Both backup and restore procedures for Anypoint Monitoring are outside the normal Gravitational backup/restore mechanism and therefore requires additional procedures.
+
+Anypoint Monitoring/Visualizer need to backup the following component's data
+
+. InfluxDB (`dias-prov-k8s-am-influxdb-comp`)
+. Cassandra (`dias-prov-k8s-am-ingestor-comp`)
+
+
+Each component has its own way to backup, and restore, the data. It is important to mentioned that the Anypoint Monitoring/Visualizer custom Stolon DB will be backup/restore by the default Gravitational mechanism and it is not considered in this document.
+
+---
+It is up to the customer to provide enough space to run the backup/restore for each of the mentioned components, considering that we will need double the space, at most. For instance, InfluxDB data directories might use 200GB and therefore the underlying disk (where the backup is meant to be written) should have 200GB+ free space.
+---
+
+=== InfluxDB
+
+All InfluxDB data/meta nodes run on the 3 nodes labeled `amv_node` and therefore we need to backup the data sitting in those 3 nodes. Data is under '/var/lib/data/influxdb' for both meta and data pods. The following steps consider that the underlying hard drive has enough space to hold the backup. The destination can be actually anything (disk, S3 bucket, etc).
+
+. `kubectl get nodes | grep amv` will provide the Anypoint nodes
+. Connect to each `amv` node and run `sudo gravity enter`
+. Under `cd /var/lib/data/influxdb` we can find the data that needs to be backup. At this point, any approach to copy that data to a secure location is acceptable. The following steps are provided just as a local backup.
+. Copy over data with `cp -a meta/. bck/meta & cp -a data/. bck/data` (`bck` is just a directory in the same location, it can be anywhere)
+. On a fresh new installation with InfluxDB pods already running we will restore data with `cp -a bck/data/. data/ & cp -a bck/meta/. meta/`
```

Comment:
Can the data be restored on any amv node in the restore environment? Is there some sort of order these need to be restarted in? These are good things to clarify I believe

Reply:
Also, is it necessary to clean /var/lib/data/influxdb in the target environment before restoring? What if there was already data under that directory?

Reply:
Description added. The script will check for existing data and ask for deletion